client/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "namex",
-  "version": "2.1.1",
+  "version": "2.1.2",
```

Comment:
Please check in lock file too.

Reply:
Weird, didn't update on an `npm i`. 
Added!

Reply:
I think you did `npm i` in root folder instead of client folder.

Reply:
Ya :( 

Removed!